pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -222,9 +222,8 @@ type PDSpec struct {
 	// +optional
 	Service *ServiceSpec `json:"service,omitempty"`
 
-	// The storageClassName of the persistent volume for PD data storage, empty string means not explicitly set
-	// and use the cluster default set by admission controller.
-	// Optionals: Defaults to the default-storage-class-name set in the tidb-operator
+	// The storageClassName of the persistent volume for PD data storage.
+	// Optionals: Defaults to the default-storage-class-name set in the tidb-operator or Kubernetes default storage classs.
```

Comment:
We shouldn't set default value for `storageClassName` in defaulting phase. 

`nil` storageClassName in PVC is a valid value that indicates the default storage class of the cluster to be used.